app/controllers/redirections_controller.rb

```diff
@@ -1,4 +1,6 @@
 class RedirectionsController < ApplicationController
+  before_action :ensure_referrer_is_not_localhost
```

Comment:
You know, in my head I was thinking something like `if redirection.save` and then handling it there, but that's not what we have. Let's just go with this. We can always change it if we need to.